src/pages/dashboard/components/EpisodeDetails.tsx

```diff
@@ -66,10 +66,38 @@ function EpisodeDetails({ episode, isInCollection = false, showDate = false }: P
     return `${((resumePosition.asMilliseconds() / duration.asMilliseconds()) * 100).toFixed(2)}%`;
   }, [episode.Duration, episode.ResumePosition]);
 
+  const anidbEpisodeAlias = (type: EpisodeTypeEnum, epTitle: string) => {
+    // TODO: This is a hack, we need update backend to make the EpisodeTypeEnum make more sense.
```

Comment:
This reply just to document this commit
>we will return 'C' for ThemeSong prefixes as default and if the title has Opening or Ending i will replace with OP or ED and keep the rest to match with AniDB 
>this logic will be same for the future if we gonna use Opening and Ending type since AniDb not gonna return song title anytime soon